api/api_public_klay.go

```diff
@@ -42,9 +44,10 @@ func NewPublicKlayAPI(b Backend) *PublicKlayAPI {
 	return &PublicKlayAPI{b}
 }
 
-// GasPrice returns a suggestion for a gas price.
+// GasPrice returns a suggestion for a gas price (baseFee * 2).
 func (s *PublicKlayAPI) GasPrice(ctx context.Context) (*hexutil.Big, error) {
 	price, err := s.b.SuggestPrice(ctx)
```

Comment:
How about chainging logic in `s.b.SuggestPrice(ctx)`?

Reply:
Eth namespace API, `GasPrice`, just call the `PublicKlayAPI.GasPrice`. So I think we need to change some logic only about setDefaults.

Reply:
Oh okayokay.
I think `SuggestPrice` function name includes `"Suggest"`, so changing logic in `SuggestPrice` function might be better in my opinion. What's your thinking with this?

Reply:
And shouldn't we also check if the current block is a hard fork or not?

Reply:
- I agree what the 'Suggest' means.
- Right, we should check the hard fork.
  - Before hard fork, It just returns the unitPrice.
  - After hard fork, It returns the BaseFee * 2.